pyproject.toml

```diff
@@ -1,3 +1,13 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project.urls]
+"Homepage" = "https://github.com/OCR-D/ocrd-webapi-implementation"
+
+[tool.setuptools]
+packages = ["ocrd_webapi"]
```

Comment:
I'm curious: Why is that information (`packages`) in both `pyproject.toml` and `setup.cfg`?

Also, can you do `pip install -e .` without a setup.py?